azerothcore/azerothcore-wotlk
# fix(Core/DB/Quest): Let Them Eat Crow

##### CHANGES PROPOSED:

-  Fixing the quest


###### ISSUES ADDRESSED:


Closes https://github.com/azerothcore/azerothcore-wotlk/issues/1637


##### TESTS PERFORMED:


- [x] Linux (Docker)

##### HOW TO TEST THE CHANGES:


.go c 107741
.quest add 11227
.additem 33238 5
Get to the quest marked location and use the item to summon the pet
Use the 5 items added to feed him
Complete the quest to see if he despawns


##### KNOWN ISSUES AND TODO LIST:


##### Target branch(es):

Master
